.github/workflows/test-windows.yml

```diff
@@ -86,6 +86,8 @@ jobs:
         choco install nasm --no-progress
         echo "C:\Program Files\NASM" >> $env:GITHUB_PATH
 
+        python -m pip install meson
+
```

Comment:
Testing, I don't find this to be necessary - https://github.com/radarhere/Pillow/commit/2d4a4de12c8902c184168cd59b5dd7f03cdb6357 / https://github.com/radarhere/Pillow/actions/runs/11400689885

Reply:
I removed this in https://github.com/fdintino/Pillow/pull/3